
(25, 'Inappropriate ioctl for device')
[nltk_data] Downloading package punkt to
[nltk_data]     /cluster/home/julievt/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
/cluster/home/julievt/master-project/src/textmining/ner/setup.py:30: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  dataset['MedicalEntity'] = dataset['MedicalEntity'].fillna('O')
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: julievt (julievt-ntnu). Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.1
wandb: Run data is saved locally in /cluster/home/julievt/master-project/wandb/run-20250213_094706-k3pqq7j5
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run dandy-oath-8
wandb: ⭐️ View project at https://wandb.ai/julievt-ntnu/ner-a-Task.TOKEN-nn-model
wandb: 🚀 View run at https://wandb.ai/julievt-ntnu/ner-a-Task.TOKEN-nn-model/runs/k3pqq7j5
wandb:                                                                                
wandb: 
wandb: Run history:
wandb: train_accuracy ▁▆▆▇▇▇▇▇██████
wandb:     train_loss █▃▃▂▂▂▂▁▁▁▁▁▁▁
wandb:        val_acc ▁▃▄▆▆▇▇███████
wandb:       val_loss █▅▃▂▂▁▁▁▁▁▁▁▂▂
wandb: 
wandb: Run summary:
wandb: train_accuracy 0.99321
wandb:     train_loss 22.38635
wandb:        val_acc 0.94609
wandb:       val_loss 61.5141
wandb: 
wandb: 🚀 View run dandy-oath-8 at: https://wandb.ai/julievt-ntnu/ner-a-Task.TOKEN-nn-model/runs/k3pqq7j5
wandb: ⭐️ View project at: https://wandb.ai/julievt-ntnu/ner-a-Task.TOKEN-nn-model
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20250213_094706-k3pqq7j5/logs
/cluster/home/julievt/master-project/src/textmining/ner/setup.py:30: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  dataset['MedicalEntity'] = dataset['MedicalEntity'].fillna('O')
wandb: Tracking run with wandb version 0.19.1
wandb: Run data is saved locally in /cluster/home/julievt/master-project/wandb/run-20250213_211824-hditmy1e
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run usual-monkey-2
wandb: ⭐️ View project at https://wandb.ai/julievt-ntnu/ner-c-Task.TOKEN-nn-model
wandb: 🚀 View run at https://wandb.ai/julievt-ntnu/ner-c-Task.TOKEN-nn-model/runs/hditmy1e
